app/__init__.py

```diff
@@ -15,11 +15,11 @@
 def create_app(config_name):
     application = Flask(__name__)
     application.wsgi_app = ProxyFix(application.wsgi_app)
-    application.config.from_object(config[config_name])
-
-    config[config_name].init_app(application)
+    application.config['DM_ENVIRONMENT'] = config_name
```

Comment:
Does this new environment variable need to be added to the `run_app` script and to the README?